addon/-private/data-view/radar/radar.js

```diff
@@ -534,6 +534,13 @@ export default class Radar {
   }
 
   get visibleBottom() {
+    // This is the case where the container of this vertical collection could have height 0px at
+    // initial render step but will be updated later. We want to return visibleBottom to be 0 rather
+    // than -1.
+    if (this.visibleTop == 0 && this._scrollContainerHeight == 0) {
+      return 0;
+    }
+
     return this.visibleTop + this._scrollContainerHeight - 1;
```

Comment:
I think this can be simplified as `return Math.max(this.visibleTop + this._scrollContainerHeight - 1, 0);`